.ci/run_tests.sh

```diff
@@ -28,3 +28,23 @@ fi
 if [ "$TEST_SUITE" == "pdk" ]; then
     TEST_NGINX_RANDOMIZE=1 prove -I. -j$JOBS -r t/01-pdk
 fi
+if [ "$TEST_SUITE" == "external_plugins" ]; then
+    set +e
+    rm -f .failed
+    cat kong-*.rockspec | grep kong- | grep -v sidecar | grep -v zipkin | grep "~" | while read line ; do
+        REPOSITORY=`echo $line | sed "s/\"/ /g" | awk -F" " '{print $1}'`
+        VERSION=`luarocks show $REPOSITORY | grep $REPOSITORY | head -1 | awk -F" " '{print $2}' | cut -f1 -d"-"`
+        REPOSITORY=`echo $REPOSITORY | sed -e 's/kong-prometheus-plugin/kong-plugin-prometheus/g'`
+        REPOSITORY=`echo $REPOSITORY | sed -e 's/kong-proxy-cache-plugin/kong-plugin-proxy-cache/g'`
+        echo $REPOSITORY
+        echo $VERSION
+        git clone https://github.com/Kong/$REPOSITORY.git --branch $VERSION --single-branch /tmp/test-$REPOSITORY
```

Comment:
Shouldn't we be testing against master, rather than the latest released version?

Reply:
Not in this repository no. We want to know that PR's and release builds of Kong have pinned their plugins at versions which are compatible.

The plugin CI is what should be testing the master of the plugin.